examples/chat/common/src/msg.rs

```diff
@@ -0,0 +1,45 @@
+use anyhow::{Context, Result};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub enum Msg {
```

Comment:
It might be worth considering a different name for the enum (perhaps Request or Command or something similar), as the name confused me a bit when I read the code for the first time due to overloading of the term 'message'.

Reply:
Agreed, renamed to `Command`.